### DISCUSSION TITLE
Frigate NVR with a newer version of go2rtc

### DISCUSSION THREAD

**Discussion by hansrijn** [blakeblackshear/frigate#7715] [Ask A Question] [answered]
the version of go2rtc that goes with frigate does not rotate270 a stream of one of my cameras. The latest version  1.7 of go2rtc did  do the trick. I could not run two docker installs together (due to my limited knowledge of docker, off course). 

Question:
Is there an easy way to define I want to run a different version of go2rtc?

**Comment by NickM-27** [comment, marked as answer]
https://docs.frigate.video/configuration/advanced#custom-go2rtc-version
